demo/specs/quickstart/gpu-test-count.yaml

```diff
@@ -0,0 +1,43 @@
+# One pod, one container asking 2 GPUs
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: gpu-test1
```

Comment:
For other tests we're using a separate namespace per test case. Should this be changed?

Reply:
yes